src-tauri/src/command/subcommand.rs

```diff
@@ -232,22 +236,40 @@ pub struct LinkWrapper {
     created_time: Option<std::time::SystemTime>,
 }
 
+#[derive(Debug, Clone, Deserialize, Serialize)]
+pub struct Properties {
+    pub title: String,
+    pub desc: Option<String>,
+}
+
```

Comment:
We have such definition in `arklib`:
https://github.com/ARK-Builders/arklib/blob/main/src/link.rs#L24